### ISSUE TITLE
Ghost Elements and shadows

### ISSUE THREAD

**Issue by rbartholomay** [uwu-loli/notify-manager-electron#2]
Hi!

inside the NotifyManager you use:

`titleBarStyle: 'hidden',`

This results in ghost title bar buttons (see image)

![Bildschirmfoto 2023-04-29 um 08 33 40](https://user-images.githubusercontent.com/1138142/235287938-e6eccb44-afc6-429c-b033-a8c170f63cd2.png)

if I comment out this line, all looks fine...

An when the Notification will be destroyed there is a shadow, while the animation runs. After the animation the shadow also destroyed.

![Bildschirmfoto 2023-04-29 um 08 45 20](https://user-images.githubusercontent.com/1138142/235288140-1d926411-55c5-48bb-8cb9-cf505cdf0916.png)

Hope the images helps...

Bye, René

**Comment by rbartholomay** [comment]
the shadow problem seems to be go away if I set `hasShadow: false` inside NotifyManager.js

```
        this.win = new BrowserWindow({
            width: 320,
            height: display.workAreaSize.height + display.workArea.y,
            skipTaskbar: true,
            alwaysOnTop: true,
            transparent: true,
            show: false,
            resizable: false,
            minimizable: false,
            fullscreenable: false,
            focusable: false,
            frame: false,
            //titleBarStyle: 'hidden',
            hasShadow:false,
            x: (this.position == 1 || this.position == 2) ? display.workAreaSize.width + display.workArea.x - 320 : 0,
            y: 0,
            webPreferences:{
                devTools: false,
                preload: __dirname + '/files/preload.js'
            }
        });
```

**Comment by ghost** [comment]
Hello.

Apparently, this bug happens on some operating systems.
In my local testing (win11 & deb11) I didn't notice this bug.
Thank you for reporting this bug. The fixes have been made in version 1.1.1 ([Release](https://github.com/fydne/notify-manager-electron/releases/tag/1.1.1))

Good day, fydne
